lib/vanagon/driver.rb

```diff
@@ -10,16 +10,18 @@
 class Vanagon
   class Driver
     include Vanagon::Utilities
-    attr_accessor :platform, :project, :target, :workdir, :verbose, :preserve
+    attr_accessor :platform, :project, :target, :workdir, :verbose, :preserve, :resume
```

Comment:
Not in this PR maybe but we should doc all of these attributes and their accessors. We got some big fat classes nowadays.